src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
+        String[] machine = { excavator.doWork(), excavator.stopWork(), bulldozer.doWork(),
+                bulldozer.stopWork(), truck.doWork(), truck.stopWork() };
+        for (String machines : machine) {
+            System.out.println(machines);
```

Comment:
Wrong implementation.

In MainApp class create Machine array with Truck, Bulldozer and Excavator and `call methods doWork() and stopWork() in a loop`.